src/stories/theme/themes.ts

```diff
@@ -17,19 +17,7 @@ export type FontDefinition = {
 
 export type FontClassDefinition = Omit<FontDefinition, 'fontSize' | 'lineHeight'>;
 
-export type ShadowDefinition = `${CSSLength} ${CSSLength} ${CSSLength} ${CSSColor}` | 'none';
-
-const Shadows = {
```

Comment:
Had replaced it with definitions within the theme because the shadows now have light mode and dark mode variants. Could keep the object and use it for light mode definitions, then override the values in the dark mode theme?

Reply:
i get what you were going for now, but one possible better approach to preserve the utility of using those shadow values in other styles would be to keep the `Shadows` object, create a new `DarkShadows` object that merges `Shadows` with overrides for the three shadows that change in dark mode, and then use that `DarkShadows` as the value in dark mode. then, you can still set `dropdown.shadow` as `Shadows.deepBelow` and override it in dark mode as `DarkShadows.deepBelow`

if we don't do that and instead keep `dropdown.shadow` defined separately, it's way easier for us to miss changing that if there ever is a change to `deepBelow` since any Figma updates/notes wouldn't address that. in general we never want to violate [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) especially in a theme file like this—any value used twice should be abstracted to a reusable variable unless there's a specific functional or referential reason for it to be separate

Reply:
Makes sense, sounds good